README.md

```diff
@@ -23,6 +26,26 @@ recommended you use the
     - [Extensions](#config-extensions)
     - [Service](#config-service)
 
+## Owners
```

Comment:
This is probably not the most important thing a newcomer to the repo needs to see. Suggest putting below getting started, etc.

Reply:
Done.